docs/en/03_Upgrading/03_Migrating_off_CWP.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Migrating off CWP
+summary: Migrate your project off the Common Web Platform v2
+---
+
+# Migrate your project off the Common Web Platform v2
+
+Until November 2021, Silverstripe Ltd maintained a specilasied version of the Silverstripe CMS project for the New Zealand Goverment and related New Zealand public agencies. This version was called _Common Web Platform_ (CWP for short).
+
+The CWP CMS release has been discontinued. CWP project who wish to continue to receive update most migrate their code base off the latest CWP release back to the regular Silverstripe CMS release.
+
+## Steps to migcate away from CWP
+
+Before you began this process you must identify which version of Silverstripe CMS Recipe you want to upgrade to. You can upgrade to [Silverstripe CMS Recipe 4.9.0](/changelogs/4.9.0/) or greater.
+
+[Review the changelog list](/changelogs/) to choose the Silverstripe CMS Recipe version you wish to upgrade to. Read the changelog carefully. It may contain additional release specific upgrade guidance beyond the steps in this guide.
+
+1. If your current composer file contains `cwp/cwp-installer`, run `composer update-recipe cwp/cwp-installer` to inline the recipe.
+2. If your current composer file contains `cwp/cwp-recipe-cms`, run `composer update-recipe cwp/cwp-recipe-cms` to inline the recipe.
+3. If your current composer file contains `cwp/cwp-recipe-core`, replace it with `silverstripe/recipe-public-cloud`.
+4. If your current composer file contains `cwp/cwp-recipe-search`, replace it with `silverstripe/recipe-public-cloud-search`.
+5. Review all your composer requirements and update the constraints so the module versions documented in the changelog for you desired Silverstripe CMS Recipe release are installable.
+6. Run a `composer update` to get the lastest tag of each module.
+
+## Questions
+
+### What constraint should I use in my requirement version?
+
+That really depends on how comfortable your organisiation is with automatically applying updates to your Silverstripe CMS project.
+
+The three broad constraint categories you can use are, in descending order of restrictiveness:
```

Comment:
I'm wondering if using 'silverstripe/framework' is the best example here? As if a project follows the steps above, they won't have this in their composer.json, they should just have recipe-cms.

Perhaps refer to something like userforms, which was included in the cwp-installer.

Reply:
I switched to userforms